test/integration/external/hca_azul_client_test.rb

```diff
@@ -89,7 +89,7 @@ def get_entries_from_response(response, key)
   end
 
   test 'should get projects' do
-    skip_if_api_down
+    # skip_if_api_down
```

Comment:
Is leaving in this commented-out code intentional?

Reply:
Nope - it is not!  Good catch.